src/containers/SignIn/index.tsx

```diff
@@ -0,0 +1,51 @@
+import React, {FC, useState} from 'react';
+import {useDispatch} from 'react-redux';
+import {useHistory} from 'react-router-dom';
+
+import {AuthContainer} from 'components';
+import {Form, FormButton, FormInput} from 'components/FormComponents';
+import {login} from 'dispatchers/app';
+import {AppDispatch} from 'types/store';
+import {formatAPIError} from 'utils/errors';
+import yup from 'utils/yup';
+
+const initialValues = {
+  email: '',
+  password: '',
+};
+
+export type FormValues = typeof initialValues;
+
+const SignIn: FC = () => {
+  const dispatch = useDispatch<AppDispatch>();
+  const history = useHistory();
+  const [errorMessage, setErrorMessage] = useState<string>('');
+
+  const handleSubmit = async ({email, password}: FormValues): Promise<void> => {
+    try {
+      const {data} = await dispatch(login({email, password}));
+      history.push(`/users/${data.pk}`);
+    } catch (error) {
+      setErrorMessage(formatAPIError(error));
+    }
+  };
+
+  const validationSchema = yup.object().shape({
+    email: yup.string().required('This field is required'),
```

Comment:
should we use `yup.string().email()...` here?

Reply:
updated